src/c++/perf_analyzer/genai-pa/genai_pa/parser.py

```diff
@@ -80,52 +99,99 @@ def add_profile_args(parser):
         required=False,
         help="The output length (tokens) to use for benchmarking LLMs. (Default: 128)",
     )
-
-
-def add_endpoint_args(parser):
     parser.add_argument(
-        "--url",
+        "--profile-export-file",
```

Comment:
This should be a required option because the metrics need it. It also should have a default for better ease-of use.

Reply:
Suggestion on file name?

Reply:
`profile_export.json`

Reply:
I believe PA errors if file already exists. We might want to prevent that from the top-level script by picking new names if file already exists. Could add `_#` to file before `.json`.

Reply:
Why not just "overwrite" the file (delete it in python wrapper if it exists) and let PA always write to same default? I think it's better have a consistent default and allow customization, rather than the default incrementing depending on existing files or not.

Reply:
Sure, overwriting orchestrated by this top-level script sounds good to me.

Reply:
I think users worried about persistence should manually specify a unique filename

Reply:
> This should be a required option because the metrics need it. It also should have a default for better ease-of use.

Why is this a required option? I understand that PA->GenAI-PA needs that, but does the user need to tell the script that file name? We could easily just set it internally? Is this also serving as the final data output file?

Reply:
> I understand that PA->GenAI-PA needs that

This is what I meant by "required". It doesn't need to be specified by the user, hence my comment about having a default value.